README.md

```diff
@@ -186,3 +186,123 @@ Add hosts to `/etc/hosts`.
+
+This was done to have ability to run code against environment close deployed one or simply test it out.
+
+Example of usage
+
+```bash
+PYTHON_ENV=docker inv python.run-python --command="--version"
+```
+
+#### run_python
+
+Run python command depending on `PYTHON_ENV` variable(`docker` or `local`).
+
+Settings:
+* `python_entry` python entry command (Default: `python`)
+* `python_service` python service name (Default: `web`)
+* `python_service_params` params for docker (Default: `--rm`)
+
+### django
+
+#### manage
+
+Run `manage.py` with specified command.
+
+This command also handle starting of required services and waiting DB to
+be ready.
+
+Requires [django_probes](https://github.com/painless-software/django-probes#basic-usage)
+
+#### makemigrations
+
+Run `makemigrations` command and chown created migrations(only for docker env).
+
+#### check_new_migrations
+
+Check if there is new migrations or not.
+
+#### migrate
+
+Run `migrate` command.
+
+Settings:
+* `django_migrate_command` migrate command (Default: `migrate`)
+
+#### resetdb
+
+Reset database to initial state (including test DB).
+
+Requires [django-extensions](https://django-extensions.readthedocs.io/en/latest/installation_instructions.html)
+
+
+#### createsuperuser
+
+Create superuser.
+
+Settings:
+* `default_superuser_email` default email of superuser (Default: `root@localhost`)
+* `default_superuser_username` default username of superuser (Default: `root`)
+* `default_superuser_password` default password of superuser (Default: `root`)
+
+#### run
+
+Run development web-server.
+
+Settings:
+* `runserver_docker_params` params for docker (Default: `--rm --service-ports`)
+* `runserver_command` runserver command (Default: `runserver_plus`)
+* `runserver_host` host of server (Default: `0.0.0.0`)
+* `runserver_port` port of server (Default: `8000`)
+* `runserver_params` params for runserver command (Default: `""`)
+
+#### shell
+
+Shortcut for manage.py shell command.
+
+Settings:
+* `shell_command` command to start python shell (Default: `shell_plus --ipython`)
+
+#### dbshell
+
+Open database shell with credentials from either local or dev env.
+
+### FastAPI
+
+#### run
+
+Run development web-server.
+
+Settings:
+* `fastapi_docker_params` params for docker (Default: `--rm --service-ports`)
+* `fastapi_uvicorn_command` uvicorn command (Default: `-m uvicorn`)
+* `fastapi_app` path to fastapi app (Default: `config:fastapi_app`)
+* `fastapi_host` host of server (Default: `0.0.0.0`)
+* `fastapi_port` port of server (Default: `8000`)
+* `fastapi_params` params for uvicorn (Default: `--reload`)
+
+
+### celery
+
+#### run
+
+Start celery worker.
+
+Settings:
+* `celery_local_cmd` command for celery (Default: `celery --app config.celery:app worker --beat --scheduler=django --loglevel=info`)
+* `celery_service_name` name of celery service (Default: `celery`)
+
+### open-api
+
+#### validate-swagger
+
+Check that generated open_api spec is valid.
```

Comment:
Add note that this command uses https://github.com/tfranzel/drf-spectacular and it's default validator